PocketKit/Sources/PocketKit/Home/HomeViewModel.swift

```diff
@@ -197,12 +197,17 @@ class HomeViewModel: NSObject {
 
     /// Fetch the latest data from core data and get the NSFetechedResults Controllers subscribing to updates
     func fetch() {
-        do {
-            try recentSavesController.performFetch()
-            try recomendationsController.performFetch()
-            try sharedWithYouController.performFetch()
-        } catch {
-            Log.capture(error: error)
+        // NOTE: despite HomeViewModel runs on MainActor, this call ends up on a different thread
```

Comment:
🤣  MainActor != safe.

@nzeltzer you will enjoy this one...

Reply:
🙃